src/main/kotlin/me/fornever/todosaurus/vcs/git/GitRemote.kt

```diff
@@ -2,16 +2,18 @@
 //
 // SPDX-License-Identifier: MIT
 
-package me.fornever.todosaurus.models
+package me.fornever.todosaurus.vcs.git
 
 import java.net.URI
 import java.nio.file.Path
 
-data class RepositoryModel(val url: URI, val rootPath: Path) {
+data class GitRemote(val url: URI, val rootPath: Path) {
```

Comment:
Let's name it `GitHostingRemote` or something like that: in general, Git remotes don't serve terms like `owner`. This model is specifically for hostings like GitHub or GitLab.